config/unicorn.rb

```diff
@@ -26,7 +36,7 @@
   end
 end
 
-after_fork do |_server, _worker|
+after_fork do |server, worker|
```

Comment:
- Unused block argument - `server`. You can omit all the arguments if you don't care about them.
- Unused block argument - `worker`. You can omit all the arguments if you don't care about them.